### DISCUSSION TITLE
About Tutorial data marker genes

### DISCUSSION THREAD

**Discussion by hahia** [theislab/scvelo#469] [Q&A]
Hi,
In the tutorial data

![image](https://user-images.githubusercontent.com/49113803/86133545-06c05300-bb1b-11ea-9d4f-d51aaf88919a.png)

scv.pl.velocity(adata, ['Cpe',  'Gnao1', 'Ins2', 'Adk'], ncols=2)
![image](https://user-images.githubusercontent.com/49113803/86132796-fd82b680-bb19-11ea-8f48-7864532b750a.png)

Are Cpe and Gnao1 markers for endocrine, Ins2 marker for  insulin-producing β-cells, Adk marker for ductal? 
Can you tell me where are you get those markers?

I don't find Ins2 and Adk in [Bastidas-Ponce et al. (2019)](https://doi.org/10.1242/dev.173849).

**Comment by WeilerP** [comment]
@VolkerBergen is there a biological reason to why you decided to plot these genes? Visually, I'd say Cpe is a marker for alpha, Ins2 for beta and Adk for ductal. Not sure about Gnao1.

@hahia what do you mean by not finding Ins2 and Adk?
